website/public/js/controllers/notificationCtrl.js

```diff
@@ -154,9 +154,9 @@ habitrpg.controller('NotificationCtrl',
     });
 
     // Quest invitation modal
-    $rootScope.$watch('party.quest.key && !party.quest.active && party.quest.members[user._id] == undefined', function(after, before){
-      if (after == before || after != true) return;
-      $rootScope.openModal('questInvitation');
+    $rootScope.$watch('user.party.quest.RSVPNeeded && (user.party.quest.completed == "" || user.party.quest.completed == undefined)', function(after, before){
```

Comment:
I don't think the order matters. When I tested this, I tested both cases of `user.party.quest.completed` (when it equals `""` and when it's null), and they both worked as expected.